packages/core/src/components/Accordion.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react";
+import {
+  View,
+  StyleSheet,
+  StyleProp,
+  TextStyle,
+  Pressable,
+} from "react-native";
+import Text from "./Text";
+import { withTheme } from "../theming";
+import type { IconSlot } from "../interfaces/Icon";
+import type { Theme } from "../styles/DefaultTheme";
+import { colorTypes } from "@draftbit/types";
+
+import {
+  COMPONENT_TYPES,
+  createTextProp,
+  createNumberProp,
+  createIconProp,
+  createColorProp,
+} from "@draftbit/types";
+
+type Props = {
+  openColor: colorTypes;
+  closedCloser: colorTypes;
```

Comment:
```suggestion
  closedCloser: string;
```

Reply:
done, along with these i have also replaced createTextProp with createStringProp for openColor, closedColor and caretColor

Reply:
in SEED_DATA

Reply:
I believe even though the TS type is `string`, the colors should be using `createColorProp` so that the platform knows to use the color picker UI.

For example:

https://github.com/draftbit/react-native-jigsaw/blob/eae6ad321143cf1010d960d7af759ed704a5393f/packages/core/src/components/Switch.tsx#L161-L163

https://github.com/draftbit/react-native-jigsaw/blob/eae6ad321143cf1010d960d7af759ed704a5393f/packages/core/src/components/Switch.tsx#L27

Reply:
ok done